src/views/router_views/psc_type/psc_type.njk

```diff
@@ -0,0 +1,54 @@
+{% extends "layouts/default.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% block main_content %}
+
+{% set pscTypeTitle = i18n.psc_type_title %}
+{% set pscTypeIndividual = i18n.psc_type_individual %}
+{% set pscTypeRle = i18n.psc_type_rle %}
+{% set pscTypeHint = i18n.psc_type_hint | safe %}
+{% set detailsTitleText = i18n.psc_type_details_title %}
+{% set detailsText = i18n.psc_type_details_text %}
+{% set continueText = i18n.global_confirm_continue_button %}
+
+<p class="govuk-body govuk-!-margin-bottom-7"></p>
+
+{{ govukRadios({
+  name: "pscType",
+  fieldset: {
+    legend: {
+      text: pscTypeTitle,
```

Comment:
a new variable pscTypeTitle isn't needed here can use the  i18n.psc_type_title directly.

Reply:
All updated to use the i18n.psc_type_ JSON references directly.